apps/condo/domains/user/components/UserProfileForm.tsx

```diff
@@ -83,6 +92,36 @@ export const UserProfileForm = () => {
         return router.push(RESET_PASSWORD_URL)
     }, [router])
 
+    const handleDeleteButtonClick = useCallback(async () => {
+        const profileExtraData = {
+            dv: 1,
+            sender: getClientSideSenderInfo(),
+        }
+
+        const saveName = false
```

Comment:
I think you should keep the name.

Reply:
@Alllex202, Why do you think that?

Reply:
> Why do you think that?

If we don't save the name, it will look bad in the interface.

Reply:
Wow, I didn't even think about it